### ISSUE TITLE
logs:vector-start fails to start vector container

### ISSUE THREAD

**Issue by rueschm** [dokku/dokku#4299]
`dokku logs:vector-start` runs with the following output

```
=====> Starting vector container
       Ensuring vector configuration exists
```

but actually the container fails during start. docker  logs of the failed vector container shows the following cause:

```
The argument '--watch-config <watch-config>' requires a value but none was supplied
```

**Comment by josegonzalez** [comment]
Okay I fixed this and added a boatload of tests for the logs plugin. Will merge/release once the PR passes.

Thanks for testing this out, and apologies for not catching this before release 🤦🏽 .

**Comment by josegonzalez** [comment]
Fixed, merged, and releasing in 0.22.7 now. Sorry!
